lib/node/runner/checker_ballot_transaction.go

```diff
@@ -312,9 +326,7 @@ func ValidateOp(st *storage.LevelDBBackend, source *block.BlockAccount, op opera
 		}
 		if source.Linked != "" {
 			// If it's a frozen account, everything must be withdrawn
-			var expected common.Amount
-			expected, err = source.Balance.Sub(common.BaseFee)
-			if casted.Amount != expected {
+			if source.Balance != casted.Amount {
```

Comment:
why it is changed?

Reply:
@kfangw Does not the comment `reverse it` what you said mean change the order of `source.Balance` and `casted.Amount`?